### DISCUSSION TITLE
Can you introduce about the hardware requirements?

### DISCUSSION THREAD

**Discussion by 95289527** [dkackman/chiaSWARM#13] [Q&A] [answered]
Are the machines that used to mine Ethereum suitable for use as nodes?
The configuration of these machines is very low except for the graphics card
Can multiple GPUs work at the same time, how to disable or select certain GPUs?Because the power supply may not be able to meet the full load of so many graphics cards at the same time

**Comment by dkackman** [comment, marked as answer]
I've got it running on former eth miners. I haven't specced out the lower limits but something as low as this has worked:

```
cpu: Core i5 2500 (from circa 2012)
RAM: 16GB
Free disk: 100GB (for model files)
```

> Can multiple GPUs work at the same time, how to disable or select certain GPUs?

It will use all available GPU's, one at a time, in a round robin fashion, so no need for you to select one as it won't be using more than one at a time. Concurrent GPU processing is coming.

**Reply by 95289527** [reply]
Is it possible to reduce the disk size requirements, because many machines that mine eth only have 16g disks, or can put some data in a remote file system like nfs

**Reply by dkackman** [reply]
This is possible. This code uses [huggingface diffusers](https://huggingface.co/docs/diffusers/index) library. It caches its data in `~/.cache/huggingface/`. The easiest way to offload that to a different volume is to make a symbolic link at that point to a different location. I always get the syntax somehwat worong but it should look something like this on linux:

```bash
ln -s /some/big/volume/huggingface /home/USERNAME/.cache/huggingface
```
